_posts/2014-07-31-dev-meetup-july-26-2014.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Monthly Dev Meetup July 2014"
+date: 2014-07-31 19:37
+comments: true
+author: zoras
+categories: [activities]
+---
+
+The tech scene in Kathmandu is ever growing but it is rather lacking a sense of community. Dev Meetup aims to fill this void with a regular meetup for developers where like minded individuals can gather, get to know each other and share their ideas/views.
+
+The meetup will be carried out monthly (last Saturday of the month) to cover web development topics like 'ruby', 'rails', 'javascript', 'devops', 'databases', etc.
+
+The first successful [Dev Meetup](https://www.facebook.com/events/768447783198796/){:.rnw-link} was held on July 26, 2014 at CloudFactory, the next Dev Meetup is scheduled for August 30th, 2014 at the same venue.
+
+Sign up at https://plus.google.com/events/cr78vpsi1466vv0ldj990fdgdr8 to join us. Kudos to CloudFactory for sponsoring both the events.
+
+Follow us [@ruby_nepal](http://twitter.com/ruby_nepal){:.rnw-link} for latest updates and news.
+
+If you are not sure what to expect or missed the 1st Dev Meetup, check out the pics and videos from the first one.
+
+**Pictures**
+
+![](https://pbs.twimg.com/media/BtdOPL3CMAAlPlN.jpg){: .center-image }
```

Comment:
@madhusudhan1234 Let's add `alt` text and show the alt text below the image centered as well. 
Can this be changed in the plugin to be reflected for all images?

Reply:
It seems we can make that feature like this way https://superdevresources.com/image-caption-jekyll/ , I will integrate such a way if you give me the permission :).